frontend/components/README.md

```diff
@@ -0,0 +1,37 @@
+# MarkBind Vue components
```

Comment:
Much of the content removed in https://github.com/MarkBind/markbind/pull/1248/commits/99245c49ac7c9e573cf91bfb775e3f8d87b04449 is still relevant and should be restored.